api/api.ts

```diff
@@ -34,22 +39,10 @@ app.post('/upload/:tenant', async (req: express.Request, res: express.Response)
             throw new Error('Error fetching Authentication API')
         }
 
-        let data: Buffer[] = []
         const filename: string = getFilename(req)
-
-        req.on('data', (chunk: Buffer) => data.push(chunk))
-
-        req.on('end', () => {
-            fs.writeFile(`./uploads/${filename}`, Buffer.concat(data), (err: NodeJS.ErrnoException | null) => {
-                if (err) {
-                    console.error(err)
-                    res.status(500).send(`Error saving file ${filename}`)
-                } else {
-                    console.log(`File ${filename} saved`)
-                    res.status(200).send(`File ${filename} saved.`)
-                }
-            })
-        })
+        const tenantConfig = await getDocByTenant(req.params.tenant)
```

Comment:
At this point its not necessary, but is important do make a issue to disscuss what may happen if the Mongo DB doens't return any tenant config or any "internet like" problem.

Reply:
Also, note that we have coupled a DB structure with the "Uploader Application" and need to provide the code necessary to create the basic structure for the MongoDB database.

Reply:
Issue to discuss these points was open -> #38